src/main/scala/scorex/core/network/NodeViewSynchronizer.scala

```diff
@@ -121,6 +125,8 @@ MR <: MempoolReader[TX]](networkControllerRef: ActorRef,
       mempoolReaderOpt = Some(reader)
   }
 
+  protected def viewHolderEvents: Receive = transactionEvents orElse modifierEvents orElse historyMemPoolChanges
```

Comment:
I think we should keep them all `private` or all `protected`.

Reply:
Yeah. Fair point. I'm going to make them protected.